eZ/Publish/Core/Persistence/Elasticsearch/Content/Search/Mapper/Standard.php

```diff
@@ -1,13 +1,13 @@
 <?php
 /**
- * File containing the Elasticsearch Mapper class
```

Comment:
Note that we now go for "This file is part of the ezpublish-kernel package", iirc.